### DISCUSSION TITLE
Surpress pre-commit message by choice but default on

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [renovatebot/renovate#26380] [Suggest an Idea]
### Tell us more.

Hello,

Even though I totally understand the message about the pre-commit note

> Note: The pre-commit manager in Renovate is not supported by the pre-commit maintainers or community. Please do not report any problems there, instead [create a Discussion in the Renovate repository](https://github.com/renovatebot/renovate/discussions/new) if you have any questions.

The message is totally understandable but potentially this could be misleading for developers using renovate but not adding any additional configuration to it.

We have created a centralized renovate tooling, so renovate runs but is kind of hidden from the other developers, as we centralized the configuration to control what it does and what is allowed.

The pre-commit message, as we used pre-commit, just throw them off thinking something is wrong, and nothing is wrong, just the way it works.

So, we thought we could hide it by choice.

What are your thoughts?

**Comment by rarkins** [comment]
I am supportive of being able to suppress it opt-in in cases like yourselves. Perhaps it's even possible through config tricks today already. Unfortunately the pre-commit maintainer got frustrated to the point of wanting us to _remove_ pre-commit from Renovate, so this is why we are being so cautious.

**Reply by juancarlosjr97** [reply]
Which is why I agree with the message!

**Reply by juancarlosjr97** [reply]
I had some attempts of trying to remove it on this project https://github.com/juancarlosjr97/renovate-pre-commit-pr-body-custom but I can see https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/pre-commit/index.ts#L14-L16 the definition of the `prBodyNotes`, but I can see that even if I set an empty array or a conditional, it just push them to an array using the default value as it can be seen here https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/update/pr/body/notes.ts#L15.

Do you have any idea how to do it differently?

**Reply by rarkins** [reply]
https://github.com/renovatebot/renovate/pull/27664/files

**Reply by juancarlosjr97** [reply]
Thank you @rarkins !
